### PR TITLE
feat: field explorer upgrades

### LINKED ISSUE #1654: Field explorer upgrades
To this page /wp-admin/admin.php?page=dt_utilities&tab=fields
- add field icons
- add field options icons
- add link to edit field ( opens D.T settings > Fields > Field ).

### PR DESCRIPTION
**Pull request by squigglybob**
resolves #1654

### CODE REVIEW

File: dt-core/admin/menu/tabs/tab-fields.php [hunk 1]
```diff
@@ -102,26 +102,52 @@ public function box_message( $type ) {
...
+                                <input type="hidden" name="post_type" value="<?php echo esc_attr( $post_type ); ?>" />
+                                <input type="hidden" name="field-select" value="<?php echo esc_html( $post_type . '_' . $field_key ) ?>" />
+
+                                <button type="submit" class="button" name="field_selected"><?php esc_html_e( 'Edit', 'disciple-tools' ) ?></button>
+                            </form>
+                        </td>
                         <td><?php echo esc_html( $field_value["name"] ) ?></td>
                         <td><?php echo esc_html( $field_key ) ?></td>
                         <td><?php echo esc_html( $field_value["type"] ) ?></td>
+                        <td align="center">
```
Comment (kodinkat) [inline dt-core/admin/menu/tabs/tab-fields.php:131]: `align="centre"` flagged as an obsolete attribute within my dev-env; maybe refactor to use `style="text-align: center;"`

Overall, logic seems to be holding true; with `Edit` button redirecting to corresponding field details view.

Good Job!! 👍

Comment (squigglybob) [reply dt-core/admin/menu/tabs/tab-fields.php:131]: sweet, thanks. Yeah it was showing up as red in my vscode but didn't tell me off. I'll change it over...

Comment (squigglybob) [reply dt-core/admin/menu/tabs/tab-fields.php]: @kodinkat ok, that should do it

**Comment by corsacca** [comment]
image
Brilliant. Thanks guys!

**Pull request merged**